### PR TITLE
Add detection LZCNT

### PR DESCRIPTION
**Pull request by toor1245**
https://github.com/google/cpu_features/issues/253

### CODE REVIEW

File: src/impl_x86__base_implementation.inl [hunk 1]
```diff
@@ -1755,6 +1757,7 @@ CacheInfo GetX86CacheInfo(void) {
   LINE(X86_RDRND, rdrnd, , , )                             \
   LINE(X86_DCA, dca, , , )                                 \
   LINE(X86_SS, ss, , , )                                   \
+  LINE(X86_LZCNT, lzcnt, , , )                             \
```
Comment (Mizux) [inline src/impl_x86__base_implementation.inl:1760]: nit should be placed after ADX to be order consistent with the X86FeaturesEnum declaration

Comment (toor1245) [reply src/impl_x86__base_implementation.inl]: thanks, updated

**Review by Mizux — APPROVED** [review]

**Comment by toor1245** [comment]
ping @gchatelet

**Comment by gchatelet** [comment]
I missed the update, thx for the ping.

**Pull request merged**